extensions/2.0/Vendor/EXT_feature_metadata/examples/microcosm/microcosm.gltf

```diff
@@ -0,0 +1,289 @@
+{
+  "asset": {
+    "version": "2.0"
+  },
+  "extensionsUsed": [
+    "EXT_feature_metadata"
+  ],
+  "extensions": {
+    "EXT_feature_metadata": {
+      "extras": {
+        "draftVersion": "0.0.0"
+      },
+      "featureClasses": {
+        "NDVI": {
+          "name": "Normalized difference vegetation index",
+          "semantic": "NDVI",
+          "properties": {
+            "NDVI": {
+              "name": "Normalized Difference Vegetation Index",
+              "semantic": "NDVI",
+              "elementType": "UINT8",
+              "normalized": true
```

Comment:
I'm a little hesitant about having multiple channels represent a scalar value and would prefer to use KTX2 instead of jpeg/png in situations like that.

Reply:
well does that mean this extension has a dependency on KTX2? (even if it's an optional dependency?)

Reply:
I don't think a dependency on KTX2 is required. It just means that png/jpeg textures will naturally be limited to UINT8 element types and users who want more precision should use the KTX2 extension.

> @lilleyse is the intent here to support as many types as is feasible, or do we want to limit it to the ones that are most natural (e.g. everything must be based on UINT8 values)?

I would prefer the latter. But I don't think we need to add any limitations in the spec because the textures themselves are self-limiting. If that makes any sense...?

Reply:
I definitely get the "use the supported textures to limit the types", but I still feel not saying anything is too hand-wavy.

Maybe put it like this?

Implementations must support the following property types at minimum:

* `BOOLEAN`
* `UINT8`
* `INT8` -- still easy to implement, it just depends on how you interpret the bytes
* normalized `UINT8`
* normalized `INT8`
* `ARRAY[INT8/UINT8, 1-4]`

Support for other data types is implementation-defined and may require additional extensions.